workers/tests/integrationtests/api/test_metadata_topics.py

```diff
@@ -21,7 +21,7 @@ def building_count():
     }
 
 
-def test_metadata_topic(client, response_template, building_count):
+def test(client, response_template, building_count):
```

Comment:
A test shouldn't be just called test. I would suggest to name the method according to what it actually tests. Adding a proper name to test would be a solution as well, but I don't think python tests are capable of doing that.

Reply:
renamed it back to the old names